pkgs/by-name/n2/n2-as-ninja/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  n2,
+  runCommand,
+}:
+
+let
+  mainProgram = "ninja";
+  pname = "n2-as-ninja";
+  inherit (n2) version;
+in
+runCommand "${pname}-${version}"
+  {
+    inherit pname version;
+
+    passthru.unwrapped = n2;
+
+    meta = n2.meta // {
+      inherit mainProgram;
+      descripton = "n2 as a drop-in replacement for ninja";
```

```suggestion
      description = "n2 as a drop-in replacement for ninja";
```
watch out for typos (and pay attention to CI)